packages/components/ng-uirouter-layout/examples/component/index.js

```diff
@@ -0,0 +1,12 @@
+angular
+  .module('myApp', ['ui.router', 'ui.bootstrap', 'ngUiRouterLayout'])
+  .config(($stateProvider) => {
```

Comment:
Previous examples was referring to a UMD version of components.

Here you are using some arrow functions. It won't work in some browsers.

In my opinions, all examples could fit into a single `index.html`

Reply:
Imo it is easier to read if JS is extracted from HTML, especially as the examples can become more complex. I can switch to `function`